db/migrate/20250422094347_add_visible_attributes_to_event.rb

```diff
@@ -0,0 +1,15 @@
+class AddVisibleAttributesToEvent < ActiveRecord::Migration[7.1]
+  def up
+    add_column :events, :visible_contact_attributes, :string
+
+    execute <<-SQL.squish
+      UPDATE events
+      SET visible_contact_attributes = 'all'
```

Comment:
```suggestion
      SET visible_contact_attributes = '["all"]'
```
Sonst wird der default nicht richtig gesetzt